index.js

```diff
@@ -6,6 +6,8 @@ var mkdirp     = require('mkdirp')
 var rimraf     = require('rimraf')
 var valid      = require('./lib/validators')
 var pkg        = require('./package.json')
+const pullPushable = require('pull-pushable')
+const p = pullPushable()
```

Comment:
Can we name this better than one letter? Perhaps "sincePushable"

Reply:
Also, could move the declaration of this. Out of the "import" block and into the plugin's `init`.

Reply:
Done, thanks!